src/components/ebay-chips-combobox/index.marko

```diff
@@ -0,0 +1,42 @@
+$ const {
+    options,
+    selected: inputSelected,
+    class: inputClass,
+    fluid,
+    error,
+    value,
+    disabled,
+    a11yDeleteButton = "Remove",
+    ...comboboxInput
+} = input as Omit<typeof input, `on${string}`>;
+
+<span class=["chips-combobox", fluid && "chips-combobox--fluid", error && "chips-combobox--error"] aria-disabled=disabled && "true">
+    <if(state.selected && state.selected.length)>
+        <ul class="chips-combobox__items">
+            <for|option, i| of=state.selected>
+                <li>
+                    <ebay-chip a11y-delete-button=a11yDeleteButton on-delete("handleDelete", i) disabled=disabled>
+                        ${option}
+                    </ebay-chip>
+                </li>
+            </for>
+        </ul>
+    </if>
+    <ebay-combobox
+        class=["chips-combobox__combobox", inputClass]
```

Comment:
Generally we want to put the extra classes on the root. The reason is sometimes users might want to be able to use those classes as selectors.